src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchTrialDAO.scala

```diff
@@ -282,7 +282,7 @@ class ElasticSearchTrialDAO(client: TransportClient, indexName: String, refreshM
       .prepareSearch(indexName)
       .setQuery(Ready)
       .addSort("name.keyword", SortOrder.ASC)
-      .setSize(1000)
+      .setSize(5000) // we should loop and make paginated requests; this will have to suffice for now
```

Comment:
since the initial launch of free credits, we have grown to ~1300 projects, so we are past this hardcoded limit. While we should move to a paginated query, I want to get the rest of the logging/resilience in this PR out without being blocked on more complex changes.